airflow/cli/commands/celery_command.py

```diff
@@ -131,6 +131,10 @@ def worker(args):
 
     if autoscale is None and conf.has_option("celery", "worker_autoscale"):
         autoscale = conf.get("celery", "worker_autoscale")
+        max_worker, min_worker = [int(x) for x in test.split("-")]
+        if min_worker > max_worker:
+            raise SystemExit("The option for worker autoscale should be max_concurrency,min_concurrency")
+
```

Comment:
```suggestion
    max_worker, min_worker = [int(x) for x in autoscale.split("-")]
    if min_worker > max_worker:
          raise SystemExit("The option for worker autoscale should be max_concurrency,min_concurrency")
```
This validation should be done for default configuration as well.